PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -742,7 +742,7 @@ bool SeqPair::ChangeSelectedBlock(design& caseNL) {
       anode=rand() % caseNL.GetSizeofBlocks();
     } // randomly choose a block
   }
-  if(caseNL.Blocks.at(anode).size()<=1) {return false;}
+  if(caseNL.Blocks.at(anode).size()<=1) {std::cout<<"anode size < 1"<<std::endl;return false;}
```

Comment:
Please merge master in and use spdlog here.

Reply:
There are a few other cout statements in this file. Above comment applies to all.

Reply:
This line is replaced by spdlog in master.

Reply:
I merged #497. Don't know why this branch is merged at the same time.

Reply:
Not sure why this happened automatically but all the commits in this PR were also included in #497. So this PR became obsolete the minute #497 was merged anyway.